README.md

```diff
@@ -1,33 +1,29 @@
 # Coredump Water Sensor API
 
 This project provides an API for querying sensor data of our LoRaWAN based
-water temperature sensors.  Checkout https://gfrör.li for an application using
+water temperature sensors. Check out https://gfrör.li for an application using
 the API.
 
-## Requirements
 
-* Ruby 2.5
-* Rails 5
+## API Endpoints
 
-## Installation
+### CRUD Endpoints
 
-    bin/setup
+The following endpoints are RESTful and allow full CRUD operations.
 
-## Run
+- `/api/sensors`
+- `/api/measurements`
+- `/api/sponsors`
 
-    bin/run
+### Application Endpoints
 
-## Tests
+The following endpoints are optimized for mobile applications and return
+read-only aggregated data.
 
-    bin/check
-
-## API Endpoints
-
-Base path is `/api/`.
-
-- `/sensors` (full CRUD)
-- `/measurements` (full CRUD)
-- `/sponsors` (full CRUD)
+- `/api/mobile_app/sensors`
+- `/api/mobile_app/sensors/<sensor-id>/sponsor`
+- `/api/mobile_app/sensors/<sensor-id>/daily_temperatures`
```

Comment:
Maybe also add the possible params from, to & limit here

Reply:
of course for /daily_temperatures and /hourly_temperatures instead of /sponsor

Reply:
Ah good point, I wanted to add parameters as well, but then forgot :slightly_smiling_face: